metron-platform/metron-indexing/src/main/java/org/apache/metron/indexing/dao/search/SearchRequest.java

```diff
@@ -106,4 +107,33 @@ public void setSort(List<SortField> sort) {
   public void setFacetFields(List<String> facetFields) {
     this.facetFields = facetFields;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+
+    SearchRequest that = (SearchRequest) o;
+
+    return indices != null ? indices.equals(that.indices) : that.indices == null
```

Comment:
Attempting to adhere to style guidelines.  Will revert to a more readable format.